README.md

```diff
@@ -17,18 +17,18 @@ The buildpack will do the following:
-| ------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `$BP_RUSTUP_ENABLED`      | Configure rustup to be enabled. This means that rustup will be used to install Rust. Default value is `true`. Set to false to use another Rust toolchain provider like [rust-dist](https://github.com/paketo-community/rust-dist).                                                       |
-| `$BP_RUST_TOOLCHAIN`      | Rust toolchain to install. Default `stable`. Other common values: `beta`, `nightly` or a specific versin number. Any [acceptable value for a toolchain](https://dev-doc.rust-lang.org/beta/edition-guide/rust-2018/rustup-for-managing-rust-versions.html) can be used here.             |
-| `$BP_RUST_PROFILE`        | Rust profile to install. Default `minimum`. Other acceptable values: `default`, `complete`. See [Rustup docs for profile](https://rust-lang.github.io/rustup/concepts/profiles.html).                                                                                                    |
-| `$BP_RUST_TARGET`         | Additional Rust target to install. Default ``, so nothing additional is installed. If there is no user-specified target and the build is running on the Paketo Tiny stack, then the Linux musl target is automatically added. Run `rustup target list` to see what valid targets exist.  |
-| `$BP_RUSTUP_INIT_VERSION` | Configure the version of rustup-init to install. It can be a specific version or a wildcard like `1.*`. It defaults to the latest `1.*` version.                                                                                                                                         |
-| `$BP_RUSTUP_INIT_LIBC`    | Configure the libc implementation used by the installed toolchain. Available options: `gnu` or `musl`. Defaults to `gnu` for compatiblity. You do not need to set this option with the Paketo full/base/tiny stacks. It can be used for compatibility with more exotic or custom stacks. |
+| Environment Variable      | Description                                                                                                                                                                                                                                                                                       |
+| ------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
@dmikusa just FYI you don't need to pad the spaces and the dashes. Markdown (at least GitHub's flavor of Markdown) will auto-extend and auto-fit the column widths as long as there are spaces between the `|` and the content.

So the first two rows could look like:

```
| Environment Variable | Description |
| - | - |
| `$BP_RUSTUP_ENABLED` | Configure rustup to be enabled. This means that rustup will be used to install Rust. Default value is `true`. Set to false to use another Rust toolchain provider like [rust-dist](https://github.com/paketo-community/rust-dist). |
| `$BP_RUST_TOOLCHAIN` | Rust toolchain to install. Default `stable`. Other common values: `beta`, `nightly` or a specific versin number. Any [acceptable value for a toolchain](https://dev-doc.rust-lang.org/beta/edition-guide/rust-2018/rustup-for-managing-rust-versions.html) can be used here. |
| `$BP_RUST_PROFILE` | Rust profile to install. Default `minimum`. Other acceptable values: `default`, `complete`. See [Rustup docs for profile](https://rust-lang.github.io/rustup/concepts/profiles.html). |
| `$BP_RUST_TARGET` | Additional Rust target to install. Default ``, so nothing additional is installed. If there is no user-specified target and the build is running on the Paketo Tiny or Static stack, then the Linux musl target is automatically added. Run `rustup target list` to see what valid targets exist. |
| `$BP_RUSTUP_INIT_VERSION` | Configure the version of rustup-init to install. It can be a specific version or a wildcard like `1.*`. It defaults to the latest `1.*` version. |
| `$BP_RUSTUP_INIT_LIBC` | Configure the libc implementation used by the installed toolchain. Available options: `gnu` or `musl`. Defaults to `gnu` for compatiblity. You do not need to set this option with the Paketo full/base/tiny/static stacks. It can be used for compatibility with more exotic or custom stacks. |
```

Obviously it works if you fully pad everything, and that makes it easier to read in the non-render version. But personally I prefer to rely on the markdown to auto-size, rather than trying to count manually.

Reply:
oh, nice.